addons/uom/views/uom_uom_views.xml

```diff
@@ -23,20 +23,26 @@
                              <field name="name"/>
                              <field name="category_id"/>
                              <field name="uom_type"/>
-                             <field name="factor"
-                                 digits="[42,5]"
-                                 attrs="{'invisible':[('uom_type','!=','smaller')],
-                                         'readonly':[('uom_type','=','bigger')]}"/>
-                             <field name="factor_inv"
-                                 digits="[42,5]"
-                                 attrs="{'invisible':[('uom_type','!=','bigger')],
-                                         'readonly':[('uom_type','!=','bigger')]}"/>
-                             <p attrs="{'invisible':[('uom_type','!=','smaller')]}" class="oe_grey" colspan="2">
-                                 e.g: 1 * (reference unit) = ratio * (this unit)
-                             </p>
-                             <p attrs="{'invisible':[('uom_type','!=','bigger')]}" class="oe_grey" colspan="2">
-                                 e.g: 1 * (this unit) = ratio * (reference unit)
-                             </p>
+                             <label for="factor"
+                                attrs="{'invisible':[('uom_type','!=','smaller')]}"/>
+                            <div attrs="{'invisible':[('uom_type','!=','smaller')]}" class="o_row">
+                                <field name="factor"
+                                    digits="[42,5]"
+                                    attrs="{'readonly':[('uom_type','=','bigger')]}"/>
+                                <span class="oe_grey">
+                                    e.g: 1 * (reference unit) = ratio * (this unit)
```

Comment:
can you remove the spaces to have a wider input: e.g: 1*(reference unit)=ratio*(this unit)